app/assets/tailwind/components/actiontext.css

```
@@ -1,311 +1,515 @@
+lexxy-toolbar {
+  background-color: var(--card);
+  border-block-end: 1px solid var(--border);
+  color: inherit;
+  display: flex;
+  font-size: inherit;
+  inset-block-start: 0;
+  margin: 0;
+  max-inline-size: 100%;
+  padding: 0.25rem;
+  position: relative;
+  position: sticky;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

position is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)